get-secretmanager-secrets/README.md

```diff
@@ -0,0 +1,133 @@
+# get-secretmanager-secrets
+
+This action fetches secrets from [Secret Manager][sm] and makes then available
+to later build steps via outputs. This is useful when you want Secret Manager to
+be the source of truth for secrets in your organization, but you need access to
+those secrets in build steps.
+
+Secrets that are successfully fetched are set as output variables and can be
+used in subsequent actions. After a secret is accessed, its value is added to
+the mask of the build to reduce the chance of it being printed or logged by
+later steps.
+
+
+## Prerequisites
+
+-   This action requires Google Cloud credentials that are authorized to access
+    the secrets being requested. See the Authorization section below for more
+    information.
+
+
+## Usage
+
+```yaml
+steps:
+- id: secrets
+  uses: GoogleCloudPlatform/github-actions/get-secretmanager-secrets@master
+  with:
+    secrets: |-
+      token:my-project/docker-registry-token
+
+# Example of using the output
+- id: publish
+  uses: foo/bar@master
+  env:
+    TOKEN: ${{steps.secrets.outputs.token}}
+```
+
+
+## Inputs
+
+-   `secrets`: (Required) The list of secrets to access and inject into the
+    environment. Due to limitations with GitHub Actions inputs, this is
+    specified as a string.
+
+    You can specify multiple secrets by putting each secret on its own line:
+
+    ```yaml
+    secrets: |-
+      output1:my-project/my-secret1
+      output2:my-project/my-secret2
+    ```
+
+    Secrets can be referenced using the following formats:
+
+    ```text
+    # Long form
+    projects/<project-id>/secrets/<secret-id>/versions/<version-id>
+
+    # Long form - "latest" version
+    projects/<project-id>/secrets/<secret-id>
+
+    # Short form
+    <project-id>/<secret-id>/<version-id>
+
+    # Short form - "latest" version
+    <project-id>/<secret-id>
+    ```
+
+-   `credentials`: (Optional) [Google Service Account JSON][sa] credentials,
+    typically sourced from a [GitHub Secret][gh-secret]. If unspecified, other
+    authentication methods are attempted.
+
+
+## Outputs
+
+Each secret is prefixed with an output name. The secret's resolved access value
+will be available at that output in future build steps.
+
+
+## Authorization
+
+There are a few ways to authenticate this action. The caller must have
+permissions to access the secrets being requested.
+
+### Via the setup-gcloud action
+
+You can provide credentials using the [setup-gcloud][setup-gcloud] action:
```

Comment:
@sethvargo, is this a typo? I see two `[setup-gcloud][setup-gcloud]`; perhaps you were aiming for a link instead?